### PR TITLE
RHPAM-2980 Update timer of SLA

### PR DESCRIPTION
**Pull request by krisv**

### CODE REVIEW

File: jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/admin/commands/ListTimersCommand.java [hunk 1]
```diff
@@ -100,6 +101,13 @@ private TimerInstanceImpl buildTimer(org.jbpm.process.instance.timer.TimerInstan
     
     protected void processNodeInstance(TimerManager tm, NodeInstanceContainer container, List<TimerInstance> timers) {
     	for (NodeInstance nodeInstance : container.getNodeInstances()) {
+        	if (((NodeInstanceImpl) nodeInstance).getSlaTimerId() != -1) {
+    			org.jbpm.process.instance.timer.TimerInstance timer = tm.getTimerMap().get(((NodeInstanceImpl) nodeInstance).getSlaTimerId());
```
Comment (gmunozfe) [inline jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/admin/commands/ListTimersCommand.java:105]: Review indentation

**Comment by elguardian** [comment]
some minor changes requested by QE. For the rest looks ok to me.

**Comment by krisv** [comment]
@elguardian @gmunozfe test fixed, indentation adjusted and rebased

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/impl/admin/ProcessInstanceAdminServiceImplTest.java [hunk 2]
```diff
@@ -389,7 +390,31 @@ public void testUpdateTimer() throws Exception {
         processInstanceId = null;
     }
     
-    
+    @Test(timeout=10000)
+    public void testListSLATimer() throws Exception {
+        processInstanceId = processService.startProcess(deploymentUnit.getIdentifier(), "UserTaskWithSLAOnTask");
+        assertNotNull(processInstanceId);
+        long scheduleTime = System.currentTimeMillis();
```
Comment (gmunozfe) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/impl/admin/ProcessInstanceAdminServiceImplTest.java:397]: This is not used

File: jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/SLAComplianceTest.java [hunk 3]
```diff
@@ -244,6 +248,70 @@ public void testSLAonUserTaskMet() throws Exception {
...
+        ksession.dispose();
+    }
+    
+    class TimerIdListener extends DefaultProcessEventListener {
+
+        private long timerId = -1;
+        private CountDownLatch latch;
+        
+        public TimerIdListener(CountDownLatch latch) {
+        	this.latch = latch;
```
Comment (gmunozfe) [inline jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/SLAComplianceTest.java:295]: It seems there's still a tab here

**Pull request merged**